Update hardware-and-software-requirements-for-biztalk-server-2020.md (#1006)
MicrosoftDocs/biztalk-docs: biztalk/install-and-config-guides/hardware-and-software-requirements-for-biztalk-server-2016.md

> |               Microsoft SQL Server                |                                                                                                     <ul><li>Microsoft SQL Server 2016</li><li>Microsoft SQL Server 2014 SP1</li></ul>                                                                                                     | Required for BizTalk Server Runtime, EDI, and BAM. <br/><br/>For optimal performance, Microsoft recommends the Enterprise Edition of SQL Server. To fully use the BizTalk Server SDK, or deploy BizTalk Server applications from Visual Studio, install the SQL Server Development Tools. <br/><br/>Other considerations: <ul><li>To use SQL Server AlwaysOn Availability Groups (AG), use SQL Server 2016 (recommend SP3). Only SQL Server 2016 AlwaysOn AG supports MSDTC; MSDTC is required by BizTalk. BizTalk Server 2016 CU5 or above is also recommended for SQL AlwaysOn AG</li><li>BAM real-time aggregation (RTA) is not supported in the Standard Edition of SQL Server. To use BAM RTA, install SQL Server Enterprise Edition.</li><li>Using SQL Server Express Edition is not recommended. The Express edition does not include certain features needed by BizTalk Server.</li><li>BizTalk Server supports all case-sensitive and case-insensitive SQL Server collations except for binary collations. Binary collations are not supported.</li></ul> |

Reviewer: Removing HTML and fixing other formatting:

```suggestion
| Microsoft SQL Server | - Microsoft SQL Server 2016 <br>- Microsoft SQL Server 2014 SP1 | Required for BizTalk Server Runtime, EDI, and BAM. <br><br>For optimal performance, Microsoft recommends the Enterprise Edition of SQL Server. To fully use the BizTalk Server SDK, or deploy BizTalk Server applications from Visual Studio, install the SQL Server Development Tools. <br><br>Other considerations: <br>- To use SQL Server AlwaysOn Availability Groups (AG), use SQL Server 2016 (SP3 recommended). Only SQL Server 2016 AlwaysOn AG supports MSDTC, which is required by BizTalk. BizTalk Server 2016 CU5 or later is also recommended for SQL AlwaysOn AG <br>- BAM real-time aggregation (RTA) is not supported in SQL Server Standard Edition. To use BAM RTA, install SQL Server Enterprise Edition. <br>- Using SQL Server Express Edition is not recommended. The Express edition doesn't include certain features needed by BizTalk Server. <br>- BizTalk Server supports all case-sensitive and case-insensitive SQL Server collations except for binary collations. Binary collations aren't supported. |
```

> |                      WinSCP                       |                                                                        WinSCP automation.zip library required to use the SFTP adapter                                                        |                                             BizTalk requires certain WinSCP version depending on which BizTalk cumulative update that is installed. For WinSCP version details, see [SFTP adapter documentation](../core/sftp-adapter.md).                                                                                                                                                                                                                                                            |

Reviewer: ```suggestion
| WinSCP | WinSCP automation.zip library required to use the SFTP adapter | BizTalk requires a specific WinSCP version based on the installed BizTalk cumulative update. For WinSCP version details, see [SFTP adapter documentation](../core/sftp-adapter.md). |
```
